web/src/components/dashboard/SidebarCardFundingNudge.test.tsx

```diff
@@ -57,6 +74,9 @@ describe("<SidebarCardFundingNudge />", () => {
       });
 
       fireEvent.click(screen.getByTestId("cta-funding-nudge"));
+      expect(
```

Comment:
[Dust] Maybe update test name to indicate analytics now being included?

Reply:
pebble - it should be a separate test, each test should assert a single "thing"